src/commands/play-youtube-video.ts

```diff
@@ -1,9 +1,11 @@
-import YouTube from "youtube-sr";
-import ytdl from "ytdl-core";
+// import YouTube from "youtube-sr";
+//import ytdl from "ytdl-core";
```

Comment:
space after //

Reply:
or just remove those import comments if we're not using

Reply:
this file is effed and left all of the comments in for chip to take a look and add some more functionality back in. Suppose this wasn't actually ready for a PR yet but wanted to get some eyes on it.

Reply:
Yeah I’ll take care of cleaning this bad boy up.